OWML.ModHelper/ModSocketConsole.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Linq;
+using System.Net;
+using System.Net.Sockets;
+using System.Text;
+using OWML.Common;
+
+namespace OWML.ModHelper
+{
+    public class ModSocketConsole : IModConsole
```

Comment:
Don't love this name. Any better ideas? And would it make sense to also rename the original ` ModConsole.cs` to specify that it logs to a file?

Reply:
Maybe ModSocketOutput and ModConsoleOutput?